var/spack/repos/builtin/packages/mercury/package.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class Mercury(CMakePackage):
+    """Mercury is a C library for implementing RPC, optimized for HPC"""
+
+    homepage = 'https://mercury-hpc.github.io/'
+    url = 'https://github.com/mercury-hpc/mercury/archive/0.9.0.tar.gz'
+    git = 'https://github.com/mercury-hpc/mercury.git'
+
+    version('develop', branch='master', submodules=True)
+    version('1.0.0', tag='v1.0.0', submodules=True, preferred=True)
+    version('0.9.0', tag='v0.9.0', submodules=True)
+
+    variant('cci', default=False, description='Use CCI for network transport')
+    variant('bmi', default=False, description='Use BMI for network transport')
+    variant('fabric', default=True, description='Use libfabric for network transport')
+    variant('selfforward', default=True,
+            description="""Mercury will short-circuit operations
+                           by forwarding to itself when possible""")
```

Comment:
Does this message contain excess whitespace when you run `spack info mercury`?

Reply:
It doesn't contain excess whitespaces but it cuts the line right after "operations", printing this:
```
Mercury will short-circuit
operations
by forwarding to itself when
possible
```
If you have a solution, let me know. I'm not familiar enough with Python.

Reply:
I corrected, actually.